### PR TITLE
[#10823] bump commons text

### LINKED ISSUE #10823: Update commons-text to 1.10.0
### Problem Description
bump apache commons-text

### Proposed Change
`<version>1.10.0</version>`__

### PR DESCRIPTION
**Pull request by JonasCir**
Fixes #10823

### CODE REVIEW

File: sormas-app/app/build.gradle [hunk 1]
```diff
@@ -111,7 +111,7 @@ dependencies {
     implementation 'commons-logging:commons-logging:1.2'
     implementation 'org.apache.commons:commons-collections4:4.4'
     implementation 'org.apache.commons:commons-lang3:3.12.0'
-    implementation 'org.apache.commons:commons-text:1.9'
+    implementation 'org.apache.commons:commons-text:1.10'
```
Comment (StefanKock) [inline sormas-app/app/build.gradle:114]: Still incorrect version, must be 1.10.0

File: sormas-base/pom.xml [hunk 2]
```diff
@@ -431,7 +431,7 @@
 			<dependency>
 				<groupId>org.apache.commons</groupId>
 				<artifactId>commons-text</artifactId>
-				<version>1.9</version>
+				<version>1.10</version>
```
Comment (StefanKock) [inline sormas-base/pom.xml:434]: Still incorrect version, must be 1.10.0

**Review by StefanKock — CHANGES_REQUESTED** [review]

**Comment by JonasCir** [comment]
this is litereally what is just did changing it to 1.10.0.

Probably squash was messed up

**Comment by JonasCir** [comment]
ah now were good

**Review by StefanKock — APPROVED** [review]
Looks now good for me.
There are a lot changes on that version  (https://commons.apache.org/proper/commons-text/changes-report.html#a1.10.0 ), I think we should wait if all tests pass.

**Pull request merged**